activesupport/lib/active_support/callbacks.rb

```diff
@@ -806,28 +806,30 @@ def reset_callbacks(name)
         def define_callbacks(*names)
           options = names.extract_options!
 
-          names.each do |name|
-            name = name.to_sym
+          ([self] + ActiveSupport::DescendantsTracker.descendants(self)).each do |target|
```

Comment:
I wonder if this pattern is worth making a `subtree` method in `ActiveSupport::DescendantsTracker` that includes `self` and `descendants`.

Reply:
Not really, but it make sense to extract this method on this module since we use this same code in more than one place.

Reply:
@rafaelfranca Sure I could do that. Should I do it as part of this PR? Any other feedback about the PR?